libsystemd-journal.cabal

```
@@ -16,11 +16,11 @@ extra-source-files:
 
 library
   exposed-modules:     Systemd.Journal
-  build-depends:       base >=4.6 && <4.15
+  build-depends:       base >=4.6 && <4.17
                      , bytestring >= 0.9.1
                      , pipes >= 4.0
                      , pipes-safe >= 2.0
-                     , text >= 0.1 && < 1.3
+                     , text >= 0.1 
                      , transformers >= 0.3
                      , unix-bytestring >= 0.3.2 && < 0.4
```

It seems to build with latest `unix-bytestring-0.4` too:
```suggestion
                     , unix-bytestring >= 0.3.2 && < 0.5
```